src/js/services/app-service.js

```diff
@@ -0,0 +1,34 @@
+import { store } from 'stores/app-store';
```

Comment:
Initially services were meant to hold real reusable business/data logic. In our webapp we decide to merge them with actions (I should do the change here at some point too). IMO it was the best place to put reusable logic even it doesn't call the server.